lib/clean.py

```diff
@@ -229,6 +229,17 @@ def clean_sexes(df: pd.DataFrame, cols: list[str]) -> pd.DataFrame:
             .str.replace(r"^m$", "male", regex=True)
             .str.replace(r"^f$", "female", regex=True)
             .str.replace(r"^unknown.*", "", regex=True)
+            .str.replace(r"^null$", "", regex=True)
+        )
+        df = standardize_from_lookup_table(
+            df,
+            col,
+            [
+                [
+                    "male"
+                ],
+                ["female", "femaale", "famale"]
+            ]
```

Comment:
Nice idea! I think all of the `str.replace` lines above can be integrated into this `standardize_from_lookup_table` call. Please add and/or modify tests though. We don't want to be over-confident about some changes and introduce bugs into the pipeline.

Reply:
Got it. I've left the code as is in this pull request with the idea that I will create a new branch for these changes.